tools/launch_instance.rb

```diff
@@ -562,24 +568,34 @@ def installation_task(task, template:, erb_binding:, template_dir: nil)
         end
       when "a_dns"
         begin
-          dyn = get_dyn
+          logger.info("Executing A_DNS task...")
+          # use route53 as default DNS solution
+          dns_solution ||= "route53"
+          # use a random DNS string if no instance prefix is given
+          instance_prefix ||= dns_component
+          #dyn = get_dyn
           ips = task[:ips]
+          ip_api = ips
+          ip_api_int = ips
+          ip_ingress = ips
+          logger.info("A_DNS->IPS: #{ips}")
 
           unless ips && !ips.empty?
             raise 'You need to specify IPs for the `a_dns` task.'
           end
-
-          dns_record = "#{task[:prefix]}.#{dns_component}"
-          logger.info "Creating '#{dns_record}' record for: #{ips.join(?,)}"
-          fqdn = dyn.dyn_replace_a_records(dns_record, ips)
+          dns_record = "#{task[:prefix]}.#{instance_prefix}"
+          logger.info "Creating #{dns_record}' record for: #{ips.join(?,)}"
+          fqdn = create_dns_records(dns_solution, instance_prefix, ip_api, ip_api_int, ip_ingress)
```

Comment:
The A record name is already saved in `dns_record`, but when calling `create_dns_records`, it is never used, but seem like creating some new records in the `create_dns_records` function.

Reply:
@jianlinliu ok, it's fixed up now.